test/integration/narrow/routes/enrich-request.test.js

```diff
@@ -271,7 +272,7 @@ describe('Enrich request test', () => {
       expect(debtDataRow[0].schemeId).toBe(1)
```

Comment:
We have a constant module which defined scheme ids -- can you replace these hardcoded values with them?

Reply:
updated